Kickstarter-iOS/Locales/ja.lproj/Localizable.strings

```diff
@@ -60,6 +60,7 @@
 "Cancel" = "キャンセル";
 "Cancel_pledge" = "プレッジを取り消す";
 "Cancel_your_pledge" = "プレッジを取り消す";
+"Cancellation_reason" = "Motivo de cancelación";
```

Comment:
Why are the Japanese strings in Spanish? 🤔

Reply:
Good question...